egs2/speechcommands/asr1/local/speechbrain_test.csv

```diff
@@ -0,0 +1,4887 @@
+ID,duration,start,stop,wav,wav_format,wav_opts,spk_id,spk_id_format,spk_id_opts,command,command_format,command_opts,transcript,transcript_format,transcript_opts
```

Comment:
There is a util script for downloading datasets from google drive link https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/asr1/scripts/utils/download_from_google_drive.sh. You can use this one for downloading. In case they change the URL, we could modify our pipeline then. It's OK that we just keep up-to-date with their latest version.

Reply:
Okay, thank you! I'll look into it tomorrow.